src/_tutorials/server/_dcat-example.html

```diff
@@ -4,6 +4,7 @@
 from sources in the example folder.
 {% endcomment -%}
 <pre class="prettyprint lang-dart">
```

Comment:
Change from `<code>` to `tag=pre+code`?

Reply:
hmm, I don't think that's required here. (This file is generated by the script.)